cli/cmd/install.go

```diff
@@ -290,6 +324,7 @@ spec:
       annotations:
         conduit.io/created-by: "{{.CliVersion}}"
     spec:
+    serviceAccount: conduit-prometheus
```

Comment:
This line needs to be indented two additional spaces. Otherwise it produces a configuration error when I apply it:

```
error: error converting YAML to JSON: yaml: line 21: mapping values are not allowed in this context
```

Reply:
Fixed